src/main/java/com/StudentManagement/utils/Messages.java

```diff
@@ -0,0 +1,21 @@
+package com.StudentManagement.utils;
+
+public class Messages {
+
+    //Generic messages
+    public static final String SUCCESS =  "SUCCESS";
+    public static final String ERROR =  "ERROR";
+    public static final String SUCCESS_CODE =  "200";
+    public static final String ERROR_CODE =  "400";
+    public static final String DATA_FETCH_FAILED = "Failed to fetch data";
+    public static final String DATA_FETCH_SUCCESS = "Data fetch success";
+
+
+    public static final String EMAIL_INVALID = "Invalid email";
+
+//    Student
+    public static final String STUDENT_REGISTER_SUCCESS = "Student Registered Successfully";
+    public static final String STUDENT_REGISTER_FAILED = "Student Registration Failed";
+    public static final String STUDENT_DELETE_SUCCESS = "Student deleted Successfully";
+    public static final String STUDENT_DELETE_FAILED = "Student deletion failed";
+}
```

Comment:
move messages into message.properties, as it can be used for Internationalization